### PR TITLE
CI: convert remaining tests using python 2 to python 3.6

### PR DESCRIPTION
**Pull request by murlock**
##### SUMMARY

Drop Python 2
Since gateway will use Python 3.7, launch tests with Python 3.7

##### ISSUE TYPE

 - Feature Pull Request
 - Bugfix Pull Request
 - Docs Pull Request

##### COMPONENT NAME
oio

##### SDS VERSION
```
master
```

### CODE REVIEW

**Comment by fvennetier** [comment]
Building with both Python 3.6 and 3.7 seems overkill. I suggest we reduce the number of test environments.

**Comment by ilitzroth** [comment]
Please keep me informed about the python version. I'm currently using 3.6 if we're moving
to 3.7 some things need to be adapted (docker files, CMakeLists.txt), but I don't expect much fallout.

**Comment by murlock** [comment]
We still target Python 3.6 at this time, I'll fix the error and reduce number of test environmment.

**Comment by fvennetier** [comment]
Now that #2096 is merged, we need to make a cascade merge, and the tests should be ok again.

File: tools/benchmark/oio-bench-object-create.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def main(threads, delay=5.0, duration=60.0):
 
 def _object_upload(ul_handler, **kwargs):
     ul_chunks = ul_handler.chunk_prep()
-    return ul_chunks.next(), 0, 'd41d8cd98f00b204e9800998ecf8427e'
+    return next(ul_chunks.next), 0, 'd41d8cd98f00b204e9800998ecf8427e'
```
Comment (AymericDu) [inline tools/benchmark/oio-bench-object-create.py:102]: I think there is one `next` too many.
```suggestion
    return next(ul_chunks), 0, 'd41d8cd98f00b204e9800998ecf8427e'
```

**Review by AymericDu — APPROVED** [review]

**Review by AymericDu — APPROVED** [review]

**Pull request merged**